hack/util.sh

```diff
@@ -74,6 +74,7 @@ function configure_os_server {
 	openshift start \
 	--write-config=${SERVER_CONFIG_DIR} \
 	--create-certs=false \
+	--dns="tcp://${API_HOST}:53" \
```

Comment:
> Those special cases won't die, because you still need them.

They don't make any sense as written.  You can override the DNS port for an API server, but only if you start `openshift start master api`, not if you do `openshift start master`, but the kube master address, that can change for both `openshift start master api` and `openshift start master`, but if you're running an `openshift start master controllers` you have to specify the listenaddress, but that setting will have no affect if you're running `openshift start master`? 

These things are virtually impossible to reason about ahead of time.  I understand why they exist (port and/or interface deconflicting), but in their current state its a bit insane.